windows/start.sh

```diff
@@ -31,4 +31,32 @@ fi
 
 echo "Starting machine $VM..."
 $DOCKER_MACHINE start $VM
-$DOCKER_MACHINE ssh $VM
+
+echo "Setting environment variables for machine $VM..."
+eval "$($DOCKER_MACHINE env --shell=bash $VM)"
+
+clear
+cat << EOF
+
+
+                        ##         .
+                  ## ## ##        ==
+               ## ## ## ## ##    ===
+           /"""""""""""""""""\___/ ===
+      ~~~ {~~ ~~~~ ~~~ ~~~~ ~~~ ~ /  ===- ~~~
+           \______ o           __/
+             \    \         __/
+              \____\_______/
+
+EOF
+echo -e "${BLUE}docker${NC} is configured to use the ${GREEN}$VM${NC} machine with IP ${GREEN}$($DOCKER_MACHINE ip $VM)${NC}"
+echo "For help getting started, check out the docs at https://docs.docker.com"
+echo
+cd
+
+docker () {
+  winpty docker $@
```

Comment:
Not exactly sure why, but the Git for Windows peeps explicitly call program.exe in their winpty convenience aliases. (https://github.com/git-for-windows/build-extra/commit/ceda40c70aa3addec73b7d36f96363c71fa67256)

It might be less likely to run into a conflict with any other docker aliases users may set. 

Do you think we should follow the {name}.exe convention?